lib/url_parser.rb

```diff
@@ -32,6 +32,7 @@ def self.try_all(url)
     GitlabURLParser.parse_to_full_url(url) ||
     BitbucketURLParser.parse_to_full_url(url) ||
     ApacheSvnUrlParser.parse_to_full_url(url) ||
+    DrupalUrlParser.parse_to_full_url(url) ||
```

Comment:
Since this is getting pretty long, WDYT about something a bit more dynamic?

``` ruby
[GithubURLParser, GitlabURLParser, ...].reduce(nil) { |_, n| r = n.send(:parse_to_full_url, url); break r if r }
```